src/engine/enginebuffer.cpp

```diff
@@ -591,14 +591,11 @@ void EngineBuffer::slotControlSeek(double fractionalPos) {
 
 // WARNING: This method runs from SyncWorker and Engine Worker
 void EngineBuffer::slotControlSeekAbs(double playPosition) {
-    auto pMacro = m_pEngineMaster->m_pMacroRecording;
-    if (pMacro != nullptr) {
-        if (pMacro->deck.isNull()) {
-            pMacro->deck = this->getGroup();
-        }
-        if (pMacro->deck == this->getGroup()) {
-            m_bHotcueJumpPending = true;
-        }
+    uint8_t expected = 1;
```

Comment:
This is a major functional change. Please do not hide it behind a commit message about cleaning up logging.

Use an enum, not magic integers.

Reply:
Woops, the commit message got messed up. And yes, I want to replace it with an enum, but I can only do so much at once - I think I actually have a TODO in the code for that as well ;)

Reply:
fixed :)